### PR TITLE
Issue #3019992 by chr.fritsch, volker.k: Add config_profile as a dev dependency

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: config/install/user.role.editor.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ permissions:
   - 'administer menu'
   - 'administer nodes'
   - 'break content lock'
+  - 'break content lock'
```
Comment (ol0lll) [inline config/install/user.role.editor.yml]: Thats duplicated

File: config/install/user.role.seo.yml [hunk 2]
```diff
@@ -22,8 +22,10 @@ permissions:
   - 'administer meta tags'
   - 'administer nodes'
   - 'administer redirects'
+  - 'administer redirects'
```
Comment (ol0lll) [inline config/install/user.role.seo.yml]: also duplicated

File: config/install/user.role.seo.yml [hunk 3]
```diff
@@ -22,8 +22,10 @@ permissions:
...
   - 'administer taxonomy'
   - 'break content lock'
+  - 'break content lock'
```
Comment (ol0lll) [inline config/install/user.role.seo.yml]: again, dupe

File: docs/development.md [hunk 4]
```diff
@@ -15,6 +15,29 @@ After that, Thunder is successfully installed. Start coding now.
 
 ----------
 
+## Update profile configuration
+
+The Thunder distributions ships the config_profile module as a dev
+dependency for easier config updates. The worklflow for updating config
```
Comment (ol0lll) [inline docs/development.md]: workflow

File: docs/development.md [hunk 5]
```diff
@@ -15,6 +15,29 @@ After that, Thunder is successfully installed. Start coding now.
...
+* Install the latest dev version of Thunder
+* Enable the Config Profile module
+  ```
+  drush en config_profile
+  ```
+* Make all your changes in the UI
+* Export your configuration
+  ```
+  drush cex
+  ```
```
Comment (ol0lll) [inline docs/development.md:32]: The configuration is exported to the chosen config_directory and simultaneously to your profile folder.

File: config/optional/core.entity_form_display.node.article.default.yml [hunk 6]
```diff
@@ -12,15 +12,16 @@ dependencies:
...
     - entity_browser
     - field_group
     - metatag
     - paragraphs
     - paragraphs_features
     - path
     - scheduler
+    - scheduler_content_moderation_integration
     - select2
+    - thunder
```
Comment (ol0lll) [inline config/optional/core.entity_form_display.node.article.default.yml:24]: Isn't that a circular dependency?

Comment (chrfritsch) [reply config/optional/core.entity_form_display.node.article.default.yml:24]: No

**Review by ol0lll — APPROVED** [review]

**Pull request merged**